spec/students_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../students'
+
+describe Student do
+  context 'When testing the Student' do
+    it 'should create a new Student when student is initialised' do
+      student = Student.new(24, 'Willy Rovo', true)
+      expect(student.age).to eq 24
+      expect(student.name).to eq 'Willy Rovo'
+    end
+  end
+end
```

Comment:
- [ Required ] Please consider adding unit tests for the following methods in the `Student` class. It is part of the project requirements that all methods in a class are tested.
  
  - `play_hooky`
  - `classroom`